addons/l10n_tr/data/template/account.tax-tr.csv

```diff
@@ -1,17 +1,112 @@
-"id","sequence","description","invoice_label","name","price_include","amount","amount_type","type_tax_use","tax_group_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","description@tr"
-"tr_vat_sale_18","11","VAT 18% (sale)","VAT %18","18%","False","18.0","percent","sale","tax_group_kdv_18","base","invoice","","KDV %18(Satış)"
-"","","","","","","","","","","tax","invoice","tr391",""
-"","","","","","","","","","","base","refund","",""
-"","","","","","","","","","","tax","refund","tr191",""
-"tr_vat_purchase_18","11","VAT 18% (purchase)","VAT %18","18%","False","18.0","percent","purchase","tax_group_kdv_18","base","invoice","","KDV %18(Alım)"
-"","","","","","","","","","","tax","invoice","tr391",""
-"","","","","","","","","","","base","refund","",""
-"","","","","","","","","","","tax","refund","tr191",""
-"tr_kdv_satis_sale_20","13","","VAT 20%","20%","False","20.0","percent","sale","tax_group_kdv_20","base","invoice","","KDV %20(Satış)"
-"","","","","","","","","","","tax","invoice","tr391",""
-"","","","","","","","","","","base","refund","",""
-"","","","","","","","","","","tax","refund","tr191",""
-"tr_kdv_satis_purchase_20","14","","VAT 20%","20%","False","20.0","percent","purchase","tax_group_kdv_20","base","invoice","","KDV %20(Alım)"
-"","","","","","","","","","","tax","invoice","tr391",""
-"","","","","","","","","","","base","refund","",""
-"","","","","","","","","","","tax","refund","tr191",""
+"id","sequence","description","invoice_label","name","price_include","amount","amount_type","children_tax_ids","type_tax_use","tax_group_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/tag_ids","description@tr"
+"tr_ers_n20","1","Export Registered Sales VAT (-20%)","-20%","EX RS -20%","False","-20","percent","","none","tr_tax_group_kdv_n20","base","invoice","","","İhracat Kayıtlı Satış KDV'si (-20%)"
+"","","","","","","","","","","","tax","invoice","tr391004","",""
+"","","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","","tax","refund","tr391004","",""
+"tr_s_wh_20_2_10","2","Sales VAT with Withholding Tax 20% (2/10)","","S WH 20% (2/10)","False","","group","tr_wh_20_2_10,tr_s_20","sale","tr_tax_group_kdv_20","","","","","Stopajlı Satış KDV'si %20 (2/10)"

```

Comment:
I would not write `S WH ... `. I would remove the S because in your case it means Sale, but in the taxonomy we use it would means Services. What I mean is that you don't specifically need to right in the name if it's a sale or purchase, as the user will not have to choose between both; on a customer invoice, only sale taxes will be available, and purchase taxes for vendor bills. Also, instead of WHV, I would remove the V, as it's kind of implied.

Reply:
Done, I didn't know that info.